src/modules/Articles.js

```diff
@@ -43,7 +42,10 @@ const Articles = {
         headers: getFromLocalStorage(),
       });
       Articles.index();
-      Popup.open('SUCCESS_MESSAGE', response.data.message);
+      store.dispatch({
+        type: 'SUBMIT',
+        payload: { status: true, message: response.data.message },
```

Comment:
minor naming thing. I think status in payload can be renamed to submitted, to it is more readable and correspond to attribute in store.